app/views/container_image/show.html.haml

```diff
@@ -1,4 +1,10 @@
 - if @display == 'containers'
  = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_nodes'.eql?(@display) && @showtype != "compare"
+ = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_groups'.eql?(@display) && @showtype != "compare"
+ = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_projects'.eql?(@display) && @showtype != "compare"
+ = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
@dclarizio, can you put something on the backlog to get this type of stuff maybe pulled into a helper?  Seems like a good candidate to refactor in some way in the future.